lib/resolvers/project.js

```diff
@@ -1,5 +1,7 @@
 const resolver = require('./base-resolver');
-const { pick } = require('lodash');
+const { pick, get } = require('lodash');
+const { generateLicenceNumber } = require('../utils');
+const moment = require('moment');
```

Comment:
Should we be using `date-fns` here?

Reply:
`moment` is fine here because it' not client-side code.

The reason for `date-fns` over `moment` is the vastly reduced bundled code size, but if it's not going over the wire to the client then it's no big deal.